.github/actions/setup/action.yml

```diff
@@ -3,20 +3,20 @@ description: 'Common setup steps for GitHub Actions'
 runs:
   using: 'composite'
   steps:
-    - name: Setup Node.js
-      uses: actions/setup-node@v4
+    - name: Install devbox
+      uses: jetify-com/devbox-install-action@v0.9.0
       with:
-        node-version: 18
+        enable-cache: 'true'
```

Comment:
FYI: Full cache was enabled in https://github.com/AmazeeLabs/silverback-template/commit/dcafe47357f149b4ec75854d075c82d689f7fe88